mapflow/mapflow.py

```diff
@@ -522,6 +530,15 @@ def show_wd_options(self, wd: WorkflowDef):
         self.dlg.clear_model_options()
         for block in wd.optional_blocks:
             self.dlg.add_model_option(block.name, checked=bool(self.settings.value(f"wd/{wd.id}/{block.name}", False)))
+        # This layout sometimes fills after checking if a project is shared, if models take longer to initialize,
+        # and needed wigets are disabled before the appearence of these checkboxes, so we do it separately after adding boxes
+        for i in range(self.dlg.modelOptionsLayout.count()):
```

Comment:
I suggest to move this as a single function to mainDialog, to keep main file shorter.